### DISCUSSION TITLE
UI Overlap On mobile view

### DISCUSSION THREAD

**Discussion by raj-rathod** [community/community#14603] [Repositories]
When i reached on github in mobile i saw an issue in repositories card . there repository type like as (Public, Private) tab hide by the star tab
![Screenshot_20220414-233447_Chrome](https://user-images.githubusercontent.com/54914917/163660617-bf4f65f2-bc55-43dd-9302-de8d79556220.jpg)

**Comment by alcere** [comment]
Hi @raj-rathod, thanks for reaching out. Can you please elaborate a bit more your issue? Also, it will be helpful to know if you are you visiting `github.com` using your browser or the mobile app?

**Reply by raj-rathod** [reply]
Hi @alcere  
> I am visiting github.com in chrome browser in android OS
